### ISSUE TITLE
feat: `onPollVote`

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2978]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "minor"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.53.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.52.1...4.53.0) (2022-12-19)

- 📝 fix #2984 [`#2984`](https://github.com/open-wa/wa-automate-nodejs/issues/2984)
- Release 4.53.0 [`e9a735e`](https://github.com/open-wa/wa-automate-nodejs/commit/e9a735e064ed2ed8b9803c5bbec2b631758286f8)
- ✨feat: `onPollVote` #2978 [`3774502`](https://github.com/open-wa/wa-automate-nodejs/commit/3774502ec7d147274a3f3a4e2b814b8b3508c4e0)
- 🥚 patches update [`f140996`](https://github.com/open-wa/wa-automate-nodejs/commit/f1409966184a8b06daed91218af22143a0c64ddc)
- updated types-only package [`81a88dc`](https://github.com/open-wa/wa-automate-nodejs/commit/81a88dc83f98907398e9791ecf7f207a4e780623)
